### PR TITLE
[IMP] web,*: update templates to owl 2

### PR DESCRIPTION
**Pull request by ged-odoo**
From now on, Owl will only accept owl 2 templates. Old templates
will be considered old, and should be retired from active service.

Note that this commit uses the simple "2" value, but you can use a more
interesting expression. For example, owl="1+1"

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by sdegueldre** [comment]
> but you can use a more interesting expression. For example, owl="1+1"

Can't find the code to handle this, in assets.js we're still only looking whether the node has the attribute or not. I suggest we force the version number to be exact though, so currently, those should be "{{1+1}}.0.0-beta-{{2*2}}", this way we are certain we don't accidentally compile perfectly valid templates after a minor update. :fish:

File: addons/mail/static/src/components/composer/composer.xml [hunk 1]
```diff
@@ -140,7 +140,7 @@
     </t>
 
     <t t-name="mail.Composer.actionButtons" owl="1">
-        <div class="o_Composer_actionButtons" t-att-class="{ 'o-composer-is-compact': composerView.isCompact }">
```
Comment (seb-odoo) [inline addons/mail/static/src/components/composer/composer.xml:143]: There seems to be some unrelated changes across the diff so make sure to check everything carefully again.
